src/database/operations/load/loadOpportunity.ts

```diff
@@ -0,0 +1,19 @@
+import { db } from '../../db';
+import { NotFoundError } from '../../../errors';
+import type { JsonResultSet, Opportunity } from '../../../types';
+
+const loadOpportunity = async (id: number): Promise<Opportunity> => {
+	const bulkUploadsQueryResult = await db.sql<JsonResultSet<Opportunity>>(
```

Comment:
This variable is incorrectly named

Reply:
Right, should probably be opportunityQueryResult or something similar

Reply:
Since we're defining much more atomic functions for db access I am actually gonna start just using plain 'ol `result`